bazel/boringssl_ppc64le.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/include/openssl/base.h b/src/include/openssl/base.h
+index 190e250ff..8538316ef 100644
+--- a/src/include/openssl/base.h
++++ b/src/include/openssl/base.h
+@@ -116,6 +116,9 @@ extern "C" {
+ #define OPENSSL_32_BIT
+ #elif defined(__myriad2__)
+ #define OPENSSL_32_BIT
++#elif defined(_ARCH_PPC64)
```

Comment:
Can you upstream this to BoringSSL? Thanks.

Reply:
Hi @htuch, we have decided to build, test and upstream these packages.

Reply:
@htuch The boringssl community isn't keen, so we will have to keep this patch.

Reply:
i think we may be able to just get back into base.h. we'd be an "unofficial" architecture but could build. we just wouldn't have assembly initially.